### PR TITLE
fix: cleanup of depreacted keys and methods

### PR DESCRIPTION
**Pull request by devendrafyle**
### Description
copilot:summary

copilot:poem

### Walkthrough
copilot:walkthrough

## Clickup
https://app.clickup.com/t/86cut6qvn

### CODE REVIEW

File: src/app/core/services/transaction.service.ts [hunk 1]
```diff
@@ -213,6 +182,7 @@ export class TransactionService {
     cacheBusterObserver: expensesCacheBuster$,
   })
   // TODO: Remove `any` type once the stats response implementation is fixed
```
Comment (Dimple16) [inline src/app/core/services/transaction.service.ts:184]: If this comment isn't valid anymore, please remove it.

File: src/app/core/services/transaction.service.ts [hunk 2]
```diff
@@ -213,6 +182,7 @@ export class TransactionService {
...
+  // Needs clean up : once we remove older my-expenses-page completely
```
Comment (Dimple16) [inline src/app/core/services/transaction.service.ts:185]: Needs clean up -> `TODO` 
`Remove/Update method once we remove older my-expenses-page completely`

**Review by Dimple16 — APPROVED** [review]
minor comments

File: src/app/core/services/transaction.service.ts (hunk 1 above)
Comment (devendrafyle) [reply src/app/core/services/transaction.service.ts]: Ok updated this

File: src/app/core/services/transaction.service.ts (hunk 2 above)
Comment (devendrafyle) [reply src/app/core/services/transaction.service.ts]: Sure, updated

**Review by Julias0 — APPROVED** [review]

**Pull request merged**